### ISSUE TITLE
Create List of Use Cases / Objectives for Website

### ISSUE THREAD

**Issue by andrewtran1995** [andrewtran1995/django_unchained#1]
Refer to this for an example:
https://selftaughtcoders.com/plan-web-application/

We should get a concrete list of use cases for the site that we want to build, and then eventually move onto perhaps wireframes.

**Comment by andrewtran1995** [comment]
@mistayam, I think the idea of creating a Circle K website is really applicable, cause we can go all sorts of different directions with it (especially w/ database stuff). Do you think you can get a list of use cases by next Friday? It'll also give us time to look at more of the tutorials on using Python on Django and so forth.

**Comment by mistayam** [comment]
@andrewtran1995 yeah! i'll try to think of some stuff! could you elaborate on use cases again? is it just functionality of each page we would want?

**Comment by andrewtran1995** [comment]
@mistayam, essentially a list of the different things we want users to be able to do on the site would be a good start. You could also do some research as to what good practice is when developing a site, I'm not sure if I'd know best practices off the top of my head (and I'm sure I don't x) )!
